src/components/Tabs.react.js

```diff
@@ -124,53 +124,62 @@ export default class Tabs extends Component {
   }
   render() {
     let EnhancedTabs;
+    let selectedTab;
+    let selectedTabContent;
 
-    if (!Array.isArray(this.props.children)) {
-      // if dcc.Tabs.children contains just one single element, it gets passed as an object
-      // instead of an array - so we put in in array ourselves!
-      this.props.children = [this.props.children];
-    }
+    if(this.props.children) {
+      if (!Array.isArray(this.props.children)) {
+        // if dcc.Tabs.children contains just one single element, it gets passed as an object
+        // instead of an array - so we put in in a array ourselves!
+        this.props.children = [this.props.children];
+      }
 
-    const amountOfTabs = this.props.children.length
+      const amountOfTabs = this.props.children.length
```

Comment:
Missing `;`

Reply:
+1, technically [we should be using semicolons](https://github.com/plotly/dev-docs/blob/master/reference/styleguides/javascript.md#semicolons)

Many more missing -- too many to mention in a review. I recommend adding 
`/*eslint semi: ["error", "always"]*/` 
to the top of this file and linting again to catch them